_src/pattern-library/sass/components/_grid.scss

```diff
@@ -0,0 +1,1125 @@
+// #GLOBALS
+// ----------------------------
+// set basic grid settings for app
+@include layout($susy-default);
+
+
+// ----------------------------
+// #UTILITY
+// ----------------------------
+%grid-container {
+    @include container();
+}
+
+// grid rows - set visual break and document flow clear (clearfix)
+%grid-row {
+    @include break();
+}
+
+%grid-gallery {
+    @include break();
+}
+
+.grid-container {
+    @extend %grid-container;
+
+    .col.first {
+        @include first();
+    }
+
+    .col.last {
+        @include last();
+    }
+
+    .col.break {
+        @include break();
+    }
+
+    .col.omega {
+        @include omega();
+    }
+}
+
+.row {
+    @extend %grid-row;
+}
+
+// break grid's columns to a new line
+.grid-break {
+    @include break;
+}
+
+// don't break the columns to a new line
+.grid-unbreak {
+    @include nobreak;
+}
+
+
+// ----------------------------
+// #CANNED
+// ----------------------------
+.grid-manual {
+    @include layout($susy-manual, clean);
+
+    // column widths
+    .col-1 {
+        @include span(1);
+    }
+
+    .col-2 {
+        @include span(2);
+    }
+
+    .col-3 {
+        @include span(3);
+    }
+
+    .col-4 {
+        @include span(4);
+    }
+
+    .col-5 {
+        @include span(5);
+    }
+
+    .col-6 {
+        @include span(6);
+    }
+
+    .col-7 {
+        @include span(7);
+    }
+
+    .col-8 {
+        @include span(8);
+    }
+
+    .col-9 {
+        @include span(9);
+    }
+
+    .col-10 {
+        @include span(10);
+    }
+
+    .col-11 {
+        @include span(11);
+    }
+
+    .col-12 {
+        @include span(full);
+    }
+
+    // shifts - before cols
+    .pre-1 {
+        @include pre(1);
+    }
+
+    .pre-2 {
+        @include pre(2);
+    }
+
+    .pre-3 {
+        @include pre(3);
+    }
+
+    .pre-4 {
+        @include pre(4);
+    }
+
+    .pre-5 {
+        @include pre(5);
+    }
+
+    .pre-6 {
+        @include pre(6);
+    }
+
+    .pre-7 {
+        @include pre(7);
+    }
+
+    .pre-8 {
+        @include pre(8);
+    }
+
+    .pre-9 {
+        @include pre(9);
+    }
+
+    .pre-10 {
+        @include pre(10);
+    }
+
+    .pre-11 {
+        @include pre(11);
+    }
+
+    // shifts - after col
+    .post-1 {
+        @include post(1);
+    }
+
+    .post-2 {
+        @include post(2);
+    }
+
+    .post-3 {
+        @include post(3);
+    }
+
+    .post-4 {
+        @include post(4);
+    }
+
+    .post-5 {
+        @include post(5);
+    }
+
+    .post-6 {
+        @include post(6);
+    }
+
+    .post-7 {
+        @include post(7);
+    }
+
+    .post-8 {
+        @include post(8);
+    }
+
+    .post-9 {
+        @include post(9);
+    }
+
+    .post-10 {
+        @include post(10);
+    }
+
+    .post-11 {
+        @include post(11);
+    }
+}
+
+// ----------------------------
+// #CANNED + RESPONSIVE
+// ----------------------------
+.grid-manual {
+
+    // CASE: small screen bp
+    .sm-col-1 {
```

Comment:
NOTE: This whole set of definitions could be made more efficient through a mixin or function, but that seemed like a clean up/optimization task once the grid gets some use.